umade-client/src/pages/CustomerUsercase/ViewListProduct.jsx

```diff
@@ -0,0 +1,129 @@
+import React, {useEffect, useState} from "react";
+import './ViewListProduct.css';
+import axios from 'axios';
+import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
+import SearchIcon from '@mui/icons-material/Search';
+import { TextField} from '@mui/material'
+import { styled, alpha } from '@mui/material/styles';
+import InputBase from '@mui/material/InputBase';
+import { purple } from '@mui/material/colors';
+const ViewListProduct = () => {
+    const [data, setData] = useState([]);
+    const [displayData, setDisplayData] = useState([]);
+    const [name, setName] = useState('');
+    const [price, setPrice] = useState(0);
+    const [amount, setAmount] = useState(0);
+    const [createdDate, setCreatedDate] = useState(0);
+    const [product, setProduct] = useState({
+        name: "",
+        price: "",
+        amount: "",
+        createdDate: ""
+
+    })
+    const detailPage = (Product) =>
+    {
+        setDisplayData([{...Product}])
+    }
+    useEffect(() => {
+        axios.get("http://localhost:8080/api/product").then((res) => setData(res.data)).catch((err) => alert(err))
+    }, []);
+    console.log(data) 
+    const Search = styled('div')(({ theme }) => ({
+        position: 'relative',
+        borderRadius: theme.shape.borderRadius,
+        backgroundColor: alpha(theme.palette.common.white, 0.15),
+        '&:hover': {
+          backgroundColor: alpha(theme.palette.common.white, 0.25),
+        },
+        marginLeft: 10,
+        width: '100%',
+        [theme.breakpoints.up('sm')]: {
+          marginLeft: theme.spacing(10),
+          marginRight: theme.spacing(13),
+          width: 'auto',
+        },
+      }));
+      
+      const SearchIconWrapper = styled('div')(({ theme }) => ({
+        padding: theme.spacing(0, 2),
+        height: '100%',
+        position: 'absolute',
+        pointerEvents: 'none',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        backgroundColor: theme.palette.primary.main,
+        borderRadius: theme.shape.borderRadius,
+      }));
+      
+      const StyledInputBase = styled(InputBase)(({ theme }) => ({
+        color: 'inherit',                
+        width: '100%',
+        border: `2px solid ${theme.palette.primary.main}`,
+        borderRadius: theme.shape.borderRadius,
+        '& .MuiInputBase-input': {
+          padding: theme.spacing(1, 1, 1, 0),
+          // vertical padding + font size from searchIcon
+          paddingLeft: `calc(1em + ${theme.spacing(7)})`,
+          transition: theme.transitions.create('width'),
+          [theme.breakpoints.up('sm')]: {
+            width: '90ch',
+            '&:focus': {
+              width: '300ch',
+            },
+            
+          },
+        },
+      }));
+    return (
+
+        <>
+        <img src="./img/Picture1.png" width={180} height={110} className='logo'/>
+
+        <nav>
+                <a href="" >Trang chủ</a>
```

Comment:
Should use react-router Link component
```suggestion
                 <Link to="/route-path">Trang chủ</Link>
```